src/containers/Storage/Vdisk/__tests__/colors.tsx

```diff
@@ -63,26 +63,39 @@ describe('VDisk state', () => {
             <>
                 <VDisk VDiskId={{Domain: 1}} />
                 <VDisk VDiskId={{Domain: 2}} VDiskState="OK" />
-                <VDisk VDiskId={{Domain: 3}}                 DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 4}}                                   FrontQueues="Green" />
+                <VDisk VDiskId={{Domain: 3}} DiskSpace="Green" />
+                <VDisk VDiskId={{Domain: 4}} FrontQueues="Green" />
                 <VDisk VDiskId={{Domain: 5}} VDiskState="OK" DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 6}} VDiskState="OK"                   FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 7}}                 DiskSpace="Green" FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 8}} VDiskState="OK" DiskSpace="Green" FrontQueues="Green" />
-            </>
+                <VDisk VDiskId={{Domain: 6}} VDiskState="OK" FrontQueues="Green" />
+                <VDisk VDiskId={{Domain: 7}} DiskSpace="Green" FrontQueues="Green" />
+                <VDisk
+                    VDiskId={{Domain: 8}}
+                    VDiskState="OK"
+                    DiskSpace="Green"
+                    FrontQueues="Green"
+                />
+                <VDisk
+                    VDiskId={{Domain: 9}}
+                    VDiskState="PDiskError"
+                    DiskSpace="Green"
+                    FrontQueues="Green"
+                />
+            </>,
         );
 
-        const [disk1, disk2, disk3, disk4, disk5, disk6, disk7, disk8] = getAllByRole('meter');
-
-        // unavailable disks display with the highest severity
-        expect(disk1.className).toMatch(/_red\b/i);
-        expect(disk2.className).not.toMatch(/_red\b/i);
-        expect(disk3.className).toMatch(/_red\b/i);
-        expect(disk4.className).toMatch(/_red\b/i);
-        expect(disk5.className).not.toMatch(/_red\b/i);
-        expect(disk6.className).not.toMatch(/_red\b/i);
-        expect(disk7.className).toMatch(/_red\b/i);
-        expect(disk8.className).not.toMatch(/_red\b/i);
+        const [disk1, disk2, disk3, disk4, disk5, disk6, disk7, disk8, disk9] =
+            getAllByRole('meter');
+
+        // unavailable disks display with the grey color
+        expect(disk1.className).toMatch(/_grey\b/i);
+        expect(disk2.className).not.toMatch(/_grey\b/i);
+        expect(disk3.className).toMatch(/_grey\b/i);
+        expect(disk4.className).toMatch(/_grey\b/i);
+        expect(disk5.className).not.toMatch(/_grey\b/i);
+        expect(disk6.className).not.toMatch(/_grey\b/i);
+        expect(disk7.className).toMatch(/_grey\b/i);
+        expect(disk8.className).not.toMatch(/_grey\b/i);
+        expect(disk9.className).not.toMatch(/_grey\b/i);
```

Comment:
those 8 disks covered all possible combinations of 3 props being defined/undefined
the new 9th doesn't add anything to this test case, this one is not about values, but rather about values presence
if you want to cover coloring in error state, better add another test case